save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/topbar/TopBarUserField.kt

```diff
@@ -97,7 +98,7 @@ private fun topBarUserField() = FC<TopBarUserFieldProps> { props ->
                             }
                         }
                     }
-                    avatar?.let { avatar ->
+                    props.userInfo?.avatar?.let {
```

Comment:
Here you use avatar from `props`

Reply:
No. I use avatar from state, but state is nullable and contains the full link. here I just check that the avatar exists

Reply:
Oh sure yes, by bad